README.md

```diff
@@ -1,10 +1,7 @@
 Trello link here: https://trello.com/b/ffCjlMki/js-tuesday
```

Comment:
I don't believe this is the correct trello link. Did you mean to use a link from ruby-monday?